core/lib/presentation/resources/image_paths.dart

```diff
@@ -190,6 +190,7 @@ class ImagePaths {
   String get icDelivering => _getImagePath('ic_delivering.svg');
   String get icError => _getImagePath('ic_error.svg');
   String get icConnectedInternet => _getImagePath('ic_connected_internet.svg');
+  String get icEmailOpen => _getImagePath('ic_email_open.svg');
```

Comment:
I don't see the uploaded .svg image file

Reply:
Please check the icon is available in the `ImagePath` before adding a new one

Reply:
done